target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c

```diff
@@ -867,6 +867,8 @@ __ag71xx_link_adjust(struct ag71xx *ag, bool update)
 	u32 cfg2;
 	u32 ifctl;
 	u32 fifo5;
+	unsigned int speed;
+	unsigned int duplex;
```

Comment:
keep the typedef styling that the rest of the file has (pick from u8, u16, u32)

Reply:
actually i see other unsigned int defined but i don't know why it has that inconsistency...

Reply:
I chose it to match types already present in `struct ag71xx`.